Docs/reference/content/reference/driver/crud/compression.md

```diff
@@ -0,0 +1,109 @@
++++
+date = "2020-01-29T22:05:03-04:00"
+title = "Compression"
+[menu.main]
+  parent = "Reference Reading and Writing"
```

Comment:
I've placed it here taking FLE as an example (which is also placed here). But Java places compression docs into `Tutorials/Connect to MongoDb`:
![image](https://user-images.githubusercontent.com/44870738/73366481-12cf8c80-42bf-11ea-95d8-e1f012f0be34.png)

Should I place this doc into `Driver/Connecting`:
![image](https://user-images.githubusercontent.com/44870738/73366557-3397e200-42bf-11ea-8009-7154d1f94404.png)
?